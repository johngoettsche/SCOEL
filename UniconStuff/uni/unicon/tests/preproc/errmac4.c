/* test error message: no #enddef */

#begdef m
line 1
line 2
