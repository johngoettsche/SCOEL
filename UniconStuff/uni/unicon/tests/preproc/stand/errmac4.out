pp: File errmac4.c; Line 3: unexpected end-of-file in #begdef
